### PR TITLE
test-case: add quotes to function arguments for func_export_pipeline

### PR DESCRIPTION
**Pull request by aiChaoSONG**
If a shell function takes a variable as its argument,
and the variable is empty, then the argument will
degrade to nothing in the function.

This patch adds quotes to function arguments to avoid
argument degrading.

Signed-off-by: Amery Song <chao.song@intel.com>

### CODE REVIEW

**Review by marc-hb — CHANGES_REQUESTED** [review]
Totally agree with the change except it is way too big. For instance is possible some function calls rely sometimes on white space inside the argument, which is bad (it should use a bash array instead) but working and this would break it.

Please resubmit with only the changes you need right now and with a separate commit per function.

**Comment by aiChaoSONG** [comment]
@marc-hb Updated, I only need the change for `func_pipeine_export`

**Review by xiulipan — DISMISSED** [review]

**Comment by marc-hb** [comment]
Looks good except you kept the older and now misleading commit subject.

**Comment by aiChaoSONG** [comment]
@xiulipan Please help to approve this PR again, as I change the misleading commit subject as Marc suggested. Once this get merged, I will test https://github.com/thesofproject/sof-test/pull/430, if the result is good, it can be merged too.

**Review by marc-hb — CHANGES_REQUESTED** [review]
>  No functional change, only add quotes.

It is a big functional change when the `$tplg` argument is empty or not defined.

**Review by marc-hb — DISMISSED** [review]

**Comment by marc-hb** [comment]
Travis builds delayed everywhere because of the required migration to travis.com https://travis-ci.community/t/builds-hang-in-queued-state/10250  shellcheck would not have been very helpful in this PR because of the number of files changed. I reviewed every line carefully.

The only failure in https://sof-ci.01.org/softestpr/PR475/build346/devicetest/ is a RTC wakeup failure on just one system, this can't be related.

Merging this to unblock #430

**Review by marc-hb — APPROVED** [review]

**Pull request merged**
